src/x/serialize/encoder_test.go

```diff
@@ -134,7 +134,10 @@ func TestTagEncoderErrorEncoding(t *testing.T) {
 	require.False(t, ok)
 
 	e.Reset()
-	tags = ident.NewTagsIterator(ident.NewTags(ident.StringTag("abc", "defg")))
+	tags = ident.NewTagsIterator(ident.NewTags(
+		ident.StringTag("abc", "defg"),
+		ident.StringTag("x", nstring(int(maxLiteralLen))),
```

Comment:
This `nstring` is weird, and only used in one place. I suggest replacing it with `strings.Repeat`.